src/com/jwetherell/algorithms/strings/StringFunctions.java

```diff
@@ -13,6 +13,10 @@ public class StringFunctions {
     private static final char SPACE = ' ';
 
     public static final String reverseWithStringConcat(String string) {
+        //Null String Handler
```

Comment:
Lines 17 includes the check to verify if the string is null or not using an 'or' statement with similar conditions. The remainder of line 18-19 handle returning the empty string if either condition is met.